booking-engine-api/operations/hotels.md

```diff
@@ -419,11 +423,21 @@ The availability and pricing is returned for each applicable combination of occu
 | `ProductIds` | array of string | optional | Unique identifiers of products which should be included into pricing calculations. |
 | `CurrencyCode` | string | optional | ISO 4217 code of the currency. If specified the prices in response will contain only single currency based on the code provided. [Supported currency codes](../guidelines/supported-currency-codes.md) |
 | `VoucherCode` | string | optional | Voucher code enabling special rate offerings. |
-| `AdultCount` | number | optional | Requested number of adults. If provided together with `ChildCount`, then `RoomOccupancyAvailabilities` will be computed only for that combination instead of all possible. If `RoomCategory` doesn’t support the given values, then the nearest applicable values are found. |
-| `ChildCount` | number | optional | Requested number of children. |
+| `OccupancyData` | array of [Occupancy Data](#occupancy-data) | required | Occupancy numbers for the reservations. |
+| ~~`AdultCount`~~ | ~~number~~ | ~~optional~~ | ~~Requested number of adults. If provided together with `ChildCount`, then `RoomOccupancyAvailabilities` will be computed only for that combination instead of all possible. If `RoomCategory` doesn’t support the given values, then the nearest applicable values are found.~~ Deprecated |
+| ~~`ChildCount`~~ | ~~number~~ | ~~optional~~ | ~~Requested number of children.~~ Deprecated |
 | `CategoryIds` | array of string | optional | Unique identifiers of categories for which should be the availability computed only. If omitted, availability of all categories is returned instead. |
 | `AvailabilityBlockId` | string | optional | Unique identifier of availability block, if present. When provided, only availability for this block is returned. |
 
+
+#### Occupancy Data
+
+| Property | Type | Contract | Description |
+| :-- | :-- | :-- | :-- |
+| `AgeCategoryId` | string | required | Age category identifier. |
```

Comment:
@progresak @EbrithilNogare If we update the docs like this, I'm not sure people will be able to figure out, where they should find the ageCategoryId.

Additionally, it might be hard for them to figure out which numbers are meant for children and which are for adults, especially for those who need to migrate from the old API.

I don't know if you already discussed this together?

Reply:
That is a pretty good question as we don't have it on Mews help. I have two possible solutions, which do you prefer?
- We can add a mention to [mews help ](https://help.mews.com/s/article/Configuring-age-categories-in-Mews?language=en_US)
- We can it here to look like this:

| `AgeCategoryId` | string | required | Age category identifier (found in url in commander: `/Commander/.../AgeCategory/Detail/{AgeCategoryId})`. |
| :-- | :-- | :-- | :-- |

Reply:
I'd go with the proposed solution from Slack:
- Add it here into the documentation. It is the fastest way.
same way like: https://mews-systems.gitbook.io/booking-engine-guide/faq#where-can-i-get-configuration-id